src/pages/models/[id]/wizard.tsx

```diff
@@ -44,5 +44,3 @@ export const getServerSideProps = createServerSideProps({
 export default function ModelEdit() {
   return <ModelWizard />;
 }
-
-ModelEdit.getLayout = (page: React.ReactElement) => <>{page}</>;
```

Comment:
Any reason why we are removing the layout here?

Reply:
we're rejiggering the training + model create pages to look more like actual pages and less like "fake" full page modals. this keeps it consistent with all the items in the + dropdown